cmd/proxy-server/main.go

```diff
@@ -132,10 +132,17 @@ func runServer(cCtx *cli.Context) error {
 		return errors.New("not all of --tls-certificate-path and --tls-private-key-path specified")
 	}
 
-	serverAttestationType, err := proxy.ParseAttestationType(serverAttestationTypeFlag)
+	// Auto-detect server attestation type if not specified
+	serverAttestationType, err := proxy.ParseAttestationType(cCtx.String("server-attestation-type"))
```

Comment:
`ParseAttestationType` will fail if `cCtx.String("client-attestation-type")` is empty

Reply:
same as above